lib/screens/more_details_page.dart

```diff
@@ -0,0 +1,182 @@
+        crossAxisAlignment: CrossAxisAlignment.end,
+        children: [
+          // add if else statement here : if username logged in matches username of question creator -> _questionArchiveButton() else call _answerQuestionButton()
+          _answerQuestionButton(),
+          SizedBox(
+            height: 15.0,
+          ),
+        ],
+      ),
+    );
+  }
+
+  FlatButton _answerQuestionButton() {
+    return FlatButton(
+      color: Color(0xFF00ADB5),
+      shape: RoundedRectangleBorder(
+        borderRadius: BorderRadius.circular(20.0),
+      ),
+      onPressed: () {
+        Navigator.push(
+          context,
+          MaterialPageRoute(
+              builder: (context) => MoreDetails()), // accept question
+        );
+      },
+      child: Container(
+          child: Padding(
+        padding: const EdgeInsets.all(8.0),
+        child: Text("Answer this question"),
+      )),
+    );
+  }
+
+  FlatButton _questionArchiveButton() {
+    return FlatButton(
+      color: Color(0xFF00ADB5),
+      shape: RoundedRectangleBorder(
+        borderRadius: BorderRadius.circular(20.0),
+      ),
+      onPressed: () {
+        Navigator.push(
+          context,
+          MaterialPageRoute(
+              builder: (context) => MoreDetails()), //archive question
+        );
+      },
+      child: Container(
+          child: Padding(
+        padding: const EdgeInsets.all(8.0),
+        child: Text("Archive this question"),
+      )),
+    );
+  }
+
+  Column _questionFieldViewer() {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      mainAxisSize: MainAxisSize.min,
+      children: <Widget>[
+        Padding(
+          padding: const EdgeInsets.only(left: 30.0, right: 30.0),
+          child: Text('Is this a question title???',
+              style: TextStyle(
+                  fontWeight: FontWeight.bold, height: 5, fontSize: 15)),
+        ),
+        Padding(
+          padding: const EdgeInsets.only(left: 30.0, right: 30.0),
+          child: Text(
+              'We move under cover and we move as one'
+              'Through the night, we have one shot to live another day'
+              'We cannot let a stray gunshot give us away'
+              'We will fight up close, seize the moment and stay in it'
+              'It’s either that or meet the business end of a bayonet'
+              'The code word is ‘Rochambeau,’ dig me?',
+              style: TextStyle()),
+        ),
+        Padding(
+          padding: const EdgeInsets.only(left: 30.0, right: 30.0),
+          child: Text('Media attached: ',
+              style: TextStyle(
+                  fontWeight: FontWeight.bold, height: 5, fontSize: 15)),
+        ),
+        Padding(
+          padding: const EdgeInsets.only(left: 30.0, right: 30.0),
+          child: Text('image_1.jpg',
+              style: TextStyle(
+                  fontSize: 12, decoration: TextDecoration.underline)),
+        )
+      ],
+    );
+  }
```

Comment:
Instead of creating Padding for all widget, I suggest to just put the whole column in a single Padding.
The following might me an alternative, test it out. Still, it will need to be scrollable, as I mention in the comment above
``` 
Padding _questionFieldViewer() {
    return Padding(
          padding: const EdgeInsets.only(left: 30.0, right: 30.0),
      child: Column(
        crossAxisAlignment: CrossAxisAlignment.start,
        mainAxisSize: MainAxisSize.min,
        children: <Widget>[
          Text('Is this a question title???',
                style: TextStyle(
                    fontWeight: FontWeight.bold, height: 5, fontSize: 15)),
           Text("The code word is ‘Rochambeau,’ dig me?The code word is ‘Rochambeau,’ dig me?The code word is ‘Rochambeau,’ dig me?",
                style: TextStyle()),
          Text('Media attached: ',
                style: TextStyle(
                    fontWeight: FontWeight.bold, height: 5, fontSize: 15)),
          Text('image_1.jpg',
                style: TextStyle(
                    fontSize: 12, decoration: TextDecoration.underline)),
        ],
      ),
    );
  }
```